app/code/Magento/InventoryAdminUi/composer.json

```diff
@@ -5,6 +5,7 @@
         "php": "~7.1.3||~7.2.0",
         "magento/framework": "*",
         "magento/module-inventory-api": "*",
```

Comment:
You need to also add a dependency from `magento/module-inventory` in `composer.json` - see the [failing test](https://travis-ci.org/magento-engcom/msi/jobs/531474826)

Reply:
Hello @willtran will you manage to fix the failing test?

Reply:
@aleron75 have just pushed new code. I stayed in the US for a while after conference and didn't have access to computer till now.